integration-tests/tests/run-workflow.integration-test.js

```diff
@@ -1,10 +1,11 @@
 const pRetry = require('p-retry')
 const { testUrl, workflowName, billingProject } = require('../utils/integration-config')
 const { withWorkspace, createEntityInWorkspace } = require('../utils/integration-helpers')
-const { click, clickable, input, signIntoTerra, waitForNoSpinners, findInGrid, navChild, findInDataTableRow } = require('../utils/integration-utils')
+const { click, clickable, findElement, input, signIntoTerra, waitForNoSpinners, findInGrid, navChild, findInDataTableRow } = require('../utils/integration-utils')
 
 
 const testEntity = { name: 'test_entity_1', entityType: 'test_entity', attributes: { input: 'foo' } }
+const findWorkflowButton = clickable({ textContains: 'Find a Workflow' })
```

Comment:
Nit: The `find` prefix feels out of place because this is a selector for a noun (`clickable`) rather than a verb (`findElement`). Maybe just name it `workflowButton` or come up with some other noun-like word.